### DISCUSSION TITLE
Access to repository's container registry

### DISCUSSION THREAD

**Discussion by mrgleba** [community/community#5680] [Codespaces]
I'm unable to ```docker pull``` from the repositorys container registry logging with the ```GITHUB_TOKEN``` (```docker login``` works ok,
on ```docker pull``` I get unauthorized) in the codespace.
Is it the token viable for such use od do I need to set up a personal token and provide it as a secret?

**Comment by asciimike** [comment]
Would you mind giving the steps listed in [our "private registry" docs](https://docs.github.com/en/codespaces/codespaces-reference/allowing-your-codespace-to-access-a-private-image-registry) a try? You can set up `GHCR_CONTAINER_REGISTRY_*` and provide a PAT and everything should work.

Longer term we are working on making this much easier, so stay tuned for more!
